.bumpversion.cfg

```diff
@@ -0,0 +1,2 @@
+[bumpversion]
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)|dev
```

Comment:
maybe let's list the files with a version here, something like 

[bumpversion:file:setup.py]
[bumpversion:file:u8timeseries/__init__.py] (to avoid the latest CI error)
[bumpversion:file:conda_recipe/u8timeseries/meta.yaml]

Reply:
Great point, applied